src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -699,21 +648,23 @@ def call(
             raise ValueError("You have to specify either input_ids or inputs_embeds")
 
         if inputs["inputs_embeds"] is None:
-            inputs["inputs_embeds"] = self.embed_tokens(inputs["input_ids"])
+            inputs_embeds = self.embed_tokens(inputs["input_ids"]) * self.embed_scale
```

Comment:
Keep using `inputs["inputs_embeds"]` everywhere otherwise it is confusing. Better to stick to one single naming for the inputs.

Reply:
OK! Leaving this open since I'll have to change it in the templates as well.